.travis.yml

```diff
@@ -2,7 +2,6 @@ language: node_js
 before_script:
   - npm install -g npm
   - npm cache clear
-  - npm install -g broccoli-cli
```

Comment:
I guess this should be enough? https://github.com/mesosphere/marathon-ui/blob/master/package.json#L36